<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfGeneralPrefWidget</class>
 <widget class="QWidget" name="WkfGeneralPrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>通用</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="statusBarCheckBox">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>显示状态栏</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>最近配置长度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="recentConfigSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="mThemeLabel">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>主题</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mThemeBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="applicationGroupBox">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>应用程序横幅</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="appTextColorLabel">
        <property name="text">
         <string>文本颜色</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="appTextLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="appTextLabel">
        <property name="text">
         <string>文本</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="appFontSpinBox">
        <property name="minimum">
         <number>6</number>
        </property>
        <property name="maximum">
         <number>24</number>
        </property>
        <property name="value">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="UtQtColorButton" name="appTextColorPushButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="appBgColorLabel">
        <property name="text">
         <string>背景颜色</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="appFontLabel">
        <property name="text">
         <string>字体大小</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="UtQtColorButton" name="appBgColorPushButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="appTextSimNameCheckBox">
        <property name="text">
         <string>使用模拟名称（如有）</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="overlayGroupBox">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>地图叠加分类标识牌</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="overlayFontLabel">
        <property name="text">
         <string>字体大小</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="overlayFontSpinBox">
        <property name="minimum">
         <number>6</number>
        </property>
        <property name="maximum">
         <number>24</number>
        </property>
        <property name="value">
         <number>12</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UtQtColorButton</class>
   <extends>QPushButton</extends>
   <header>UtQtColorButton.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>statusBarCheckBox</tabstop>
  <tabstop>recentConfigSpinBox</tabstop>
  <tabstop>mThemeBox</tabstop>
  <tabstop>appTextLineEdit</tabstop>
  <tabstop>appFontSpinBox</tabstop>
  <tabstop>appTextColorPushButton</tabstop>
  <tabstop>appBgColorPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
